packages/context-editor/package.json

```diff
@@ -51,6 +51,37 @@
 	},
 	"dependencies": {
 		"@benrbray/prosemirror-math": "^1.0.0",
+		"@codemirror/autocomplete": "^6.18.6",
```

Comment:
legacy had these at `0.20.0` but the latest codemirror is on 6 now! (and is what the original package has updated to as well)